core/src/test/scala/org/broadinstitute/dsde/rawls/mock/MockDataRepoDAO.scala

```diff
@@ -14,6 +14,7 @@ class MockDataRepoDAO(instanceName: String) extends DataRepoDAO {
     snap.id(snapshotId)
     snap.name("snapshotName")
     snap.description("snapshotDescription")
+    snap.source(java.util.List.of(new SnapshotSourceModel().dataset(new DatasetSummaryModel())))
```

Comment:
Necessary since now all tests that use `getSnapshot` will be checking the dataset.